### DISCUSSION TITLE
edu approved but hasnt come still

### DISCUSSION THREAD

**Discussion by baturax** [community/community#118064] [GitHub Education] [answered]
### Hi everyone,

yesterday it got approved 
![image](https://github.com/community/community/assets/68316278/90d7b87d-fb13-4a96-a23f-ff1f856e42b0)
but neither any mail came nor i can access pro

**Comment by moozunch** [comment, marked as answer]
it says it approved only means your application is being checked, if you are accepted you will get a congratulation email and you're account will automatically turn into a pro account, you will need wait a bit more till they are finish checking your request (usually take few days, mine 4 days but it can be faster depends on the situation), don't worry I believed you will get your benefit soon ^_____^

**Reply by baturax** [reply]
thank you sir

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education. 🎉
